src/Audit.php

```diff
@@ -48,6 +54,10 @@ public function notify($message, array $params = array())
         try {
             $this->registerErrorHandler();
 
+            if (!$this->getEnabled()) {
+                return true;
```

Comment:
You must restore error handler before return `null`

`$this->restoreErrorHandler();`

Reply:
I have moved the registerErrorhandler after the enabler check :+1: 
```
       try {
            if (!$this->getEnabled()) {
                return true;
            }
            $this->registerErrorHandler();
```
Looks ok this way ?